easybuild/easyconfigs/d/DAS_Tool/DAS_Tool-1.1.1-foss-2018b-R-3.5.1-Python-2.7.15.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'Tarball'
+
+name = 'DAS_Tool'
+version = '1.1.1'
+versionsuffix = '-R-%(rver)s-Python-%(pyver)s'
+
+homepage = 'https://github.com/cmks/DAS_Tool'
+description = """DAS Tool is an automated method that integrates the results of a flexible number of binning
+ algorithms to calculate an optimized, non-redundant set of bins from a single assembly."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/cmks/DAS_Tool/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['2a55f67b5331251d8fd5adea867cc341363fbf7fa7ed5c3ce9c7679d8039f03a']
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('R', '3.5.1', '-Python-%(pyver)s'),
+    ('Ruby', '2.6.1'),
+    ('pullseq', '1.0.2'),
+    ('prodigal', '2.6.3'),
+    ('BLAST+', '2.7.1'),
+]
+
+exts_defaultclass = 'RPackage'
+
+exts_list = [
+    (name, version, {
+        'source_tmpl': 'DASTool_%(version)s.tar.gz',
+        'source_urls': ['https://github.com/cmks/DAS_Tool/raw/%(version)s/package/'],
+        'checksums': ['8d33997baaaec00d253b2d749cf1ace004ccdea2275b763d4d0f1c969916b72b'],
+        'modulename': 'DASTool',
+    }),
+]
+
+postinstallcmds = [
+    "cd %(installdir)s; unzip db.zip",
```

Comment:
can you add something from this zip in the sanity check?

Reply:
done in 4ed0e95